js/test/shopping-list.test.js

```diff
@@ -0,0 +1,61 @@
+const expect = chai.expect;
+const should = chai.should();
+
+describe('ShoppingListItem', function () {
+
+  let newItem;
+
+  beforeEach(function () {
+    newItem = new ShoppingListItem('drugs', 'meth');
+  })
+  it('should be a class', function () {
+    expect(newItem).to.be.instanceOf(ShoppingListItem);
+  });
+
+  it('should have property "name"', function () {
+    expect(newItem).to.have.property('name');
+    expect(newItem.name).to.be.a('string');
+    expect(newItem.name.length).to.be.gt(1);
+  })
+  it('should have property "description"', function () {
+    expect(newItem).to.have.property('description');
+    expect(newItem.description).to.be.a('string');
+    expect(newItem.description.length).to.be.gt(1);
```

Comment:
You have a test that checks that the item has a description that is a string with at least one character, but you do not exercise that test by passing a "bad" item to the test.